content/zh/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -1,212 +1,102 @@
 ---
-approvers:
+reviewers:
 - erictune
 - soltysh
 - janetkuo
-title: Cron Job
-redirect_from:
```

Comment:
interesting, why do you modify these lines?

Reply:
ok, I can see you grab the file from the master branch.

Reply:
The old file is removed here. :)